src/components/Navbar/Navbar.jsx

```diff
@@ -0,0 +1,39 @@
+import { FormGroup, FormControlLabel, Hidden } from '@material-ui/core';
+import React from 'react';
+import { SearchBar, ThemeSwitch } from '../index';
+import {
+  Nav,
+  BurgerMenuButton,
+  Left,
+  Right,
+  UserProfileButton,
+} from './Navbar.components';
+
+const Navbar = () => {
+  return (
+    <header style={{ position: 'fixed', width: '100%' }}>
+      <Nav>
+        <Left>
+          <BurgerMenuButton />
+          <SearchBar />
+        </Left>
+        <Right>
+          <Hidden xsDown>
+            <FormGroup>
+              <FormControlLabel
+                control={<ThemeSwitch />}
+                label="Dark mode"
+                style={{ color: 'white' }}
+              />
+            </FormGroup>
+          </Hidden>
+          <UserProfileButton>
+            <i class="fas fa-user fa-2x" style={{ color: '#333' }}></i>
```

Comment:
I think a conflict using fontawesome or boostrap with react, is that they use the class to do work, and react uses classname, I saw that font awesome has already an implementation with react https://fontawesome.com/v5.15/how-to-use/on-the-web/using-with/react but I don't know if it's free.
I can suggest you this one https://www.npmjs.com/package/react-icons